SpotlightDownloader/Desktop.cs

```diff
@@ -1,14 +1,14 @@
 ﻿using System;
 using System.Runtime.InteropServices;
 
-namespace SharpTools
```

Comment:
For your information, the SharpTools namespace comes from this project: https://github.com/ORelio/Sharp-Tools
The SharpTools classes may contain features that seems useless for this project, because they are used elsewhere.
Feel free to remove unneeded code and rename the namespace in these classes.